bittensor_cli/src/commands/stake/add.py

```diff
@@ -351,8 +351,14 @@ async def stake_extrinsic(
     remaining_wallet_balance = current_wallet_balance
     max_slippage = 0.0
 
+    # Convert amount to a list if it's a list, otherwise use single amount for all netuids
+    amount_list = None
```

Comment:
```suggestion
    amount_list = []
```
Keep it one type. No reason to use an optional here.